### PR TITLE
LPS-165992 | Test Automation | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
[LPS-165992](https://issues.liferay.com/browse/LPS-165992) - Automate ObjectFields#CanNotOrderByAggregationFieldColumn

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 1]
```diff
@@ -1101,6 +1101,62 @@ definition {
...
 	}
 
+	@description = "LPS-156704 - Verify that the user is not able to order the object entry's table by the aggregation field column"
+	@priority = "4"
+	test CanNotOrderByAggregationFieldColumn {
+		task ("Create Custom Object and Custom Relationship with Account (via API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161891",
+				objectName = "CustomObject161891",
+				pluralLabelName = "Custom Objects 161891");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:1111]: Use the ticket number correct to add the Custom Object - [LPS-165992](https://issues.liferay.com/browse/LPS-165992)

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 2]
```diff
@@ -1101,6 +1101,62 @@ definition {
...
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObject161891",
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:1114]: Also too - [LPS-165992](https://issues.liferay.com/browse/LPS-165992)

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 3]
```diff
@@ -1101,6 +1101,62 @@ definition {
...
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Text Field",
+				fieldName = "customTextField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject161891");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject161891");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:1128]: Also too - [LPS-165992](https://issues.liferay.com/browse/LPS-165992)

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 4]
```diff
@@ -1101,6 +1101,62 @@ definition {
...
+		}
+
+		task ("Go to Custom Object > Fields tab and add Custom Aggregation Field (via UI)") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 161891");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:1134]: Also too - [LPS-165992](https://issues.liferay.com/browse/LPS-165992)

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 5]
```diff
@@ -1101,6 +1101,62 @@ definition {
...
+				fieldLabel = "Custom Aggregation",
+				fieldType = "Aggregation",
+				function = "Max",
+				relationship = "RelationshipAccount");
+		}
+
+		task ("Go to Custom Object > Add entry and Assert the sort button is not present") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customTextField",
+				objectName = "CustomObject161891",
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:1149]: Also too - [LPS-165992](https://issues.liferay.com/browse/LPS-165992)

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 6]
```diff
@@ -1101,6 +1101,62 @@ definition {
...
+				value = "Test Entry");
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject161891");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:1152]: Also too - [LPS-165992](https://issues.liferay.com/browse/LPS-165992)

**Review by andreefarias — CHANGES_REQUESTED** [review]
Hi @rafaellajordao,

The testcases were well written. Added some comments to avoid CI errors. Check and fix it. Thanks!

**Comment by rafaellajordao** [comment]
conflict resolved in this [PR#279](https://github.com/liferay-bpm-qa/liferay-portal/pull/279)

**Pull request closed without merging**